book/src/threat_model.md

```diff
@@ -39,7 +39,7 @@ This section introduces invariants maintained by Duchess using Rust's type syste
 
 ### The `'jvm` lifetime `&mut Jvm<'jvm>` is the innermost scope for local variables
 
-References to Java objects of type `J` are stored in a `Local<'jvm, J>` holder. Local references can come from the arguments to native functions or from `JvmOp::execute_with` calls. `execute_with` calls use the `'jvm`' lifetime found on the `Jvm<'jvm>` argument. This allows the `Local` to be used freely within that scope. It is therefore important that `'jvm` be constrained to the **innermost** valid scope.
+References to Java objects of type `J` are stored in a `Local<'jvm, J>` holder. Local references can come from the arguments to native functions or from `JvmOp::do_jni` calls. `do_jni` calls use the `'jvm`' lifetime found on the `Jvm<'jvm>` argument. This allows the `Local` to be used freely within that scope. It is therefore important that `'jvm` be constrained to the **innermost** valid scope.
```

Comment:
I think we can delete the below section about `Jvm::with` now since `Jvm::with` is `pub(crate)`